safe/definitions/reports/infographic.py

```diff
@@ -0,0 +1,250 @@
+}
+
+icon_elderly_displaced = {
+    'id': 'icon-elderly-displaced',
+    'path': resources_path('img', 'definitions', 'elderly.svg')
+}
+
+icon_male_displaced = {
+    'id': 'icon-male-displaced',
+    'path': resources_path('img', 'definitions', 'male.svg')
+}
+
+icon_female_displaced = {
+    'id': 'icon-female-displaced',
+    'path': resources_path('img', 'definitions', 'female.svg')
+}
+
+icon_under_5_displaced = {
+    'id': 'icon-under-5-displaced',
+    'path': resources_path('img', 'definitions', 'under_5.svg')
+}
+
+icon_over_60_dsplaced = {
+    'id': 'icon-over-60-displaced',
+    'path': resources_path('img', 'definitions', 'over_60.svg')
+}
+
+icon_disabled_displaced = {
+    'id':'icon-disabled-displaced',
+    'path': resources_path('img', 'definitions', 'disabled.svg')
+}
+
+icon_child_bearing_age_displaced = {
+    'id': 'icon-child-bearing-age-displaced',
+    'path': resources_path('img', 'definitions', 'child_bearing.svg')
+}
+
+icon_pregnant_displaced = {
+    'id': 'icon-pregnant-displaced',
+    'path': resources_path('img', 'definitions', 'pregnant.svg')
+}
+
+icon_lactating_displaced = {
+    'id': 'icon-lactating-displaced',
+    'path': resources_path('img', 'definitions', 'pregnant_lactating.svg')
+}
+
+icon_minimum_needs_rice = {
+    'id': 'icon-minimum-needs-rice',
+    'path': resources_path('img', 'definitions', 'rice.svg')
+}
+
+icon_minimum_needs_drinking_water = {
+    'id': 'icon-minimum-needs-drinking-water',
+    'path': resources_path('img', 'definitions', 'drinking_water.svg')
+}
+
+icon_minimum_needs_clean_water = {
+    'id': 'icon-minimum-needs-clean-water',
+    'path': resources_path('img', 'definitions', 'clean_water.svg')
+}
+
+icon_minimum_needs_family_kit = {
+    'id': 'icon-minimum-needs-family-kit',
+    'path': resources_path('img', 'definitions', 'family_kits.svg')
+}
+
+icon_minimum_needs_toilet = {
+    'id': 'icon-minimum-needs-toilet',
+    'path': resources_path('img', 'definitions', 'toilets.svg')
+}
+
+icon_additional_rice = {
+    'id': 'icon-additional-rice',
+    'path': resources_path('img', 'definitions', 'rice.svg')
+}
+
+icon_hygiene_pack = {
+    'id': 'icon-hygiene-pack',
+    'path': resources_path('img', 'definitions', 'hygiene_pack.svg')
+}
+
+image_item_elements = [
+    inasafe_logo_white,
+    icon_overview,
+    population_chart,
+    icon_affected,
+    icon_displaced,
+    icon_infant_displaced,
+    icon_child_displaced,
+    icon_adult_displaced,
+    icon_elderly_displaced,
+    icon_male_displaced,
+    icon_female_displaced,
+    icon_under_5_displaced,
+    icon_over_60_dsplaced,
+    icon_disabled_displaced,
+    icon_child_bearing_age_displaced,
+    icon_pregnant_displaced,
+    icon_lactating_displaced,
+    icon_minimum_needs_rice,
+    icon_minimum_needs_drinking_water,
+    icon_minimum_needs_clean_water,
+    icon_minimum_needs_family_kit,
+    icon_minimum_needs_toilet,
+    icon_additional_rice,
+    icon_hygiene_pack
+]
```

Comment:
__[flake8]__
* [W292] no newline at end of file

<!-- Comment by [SideCI](https://sideci.com) -->